### DISCUSSION TITLE
Point Blank + LTBEEF combined (Maybe?)

### DISCUSSION THREAD

**Discussion by schoolexploitkid** [3kh0/ext-remover#1210] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
Can we inject code with XSS-attack, such as LTBEEF, into other admin forced ext. such as Lockdown Browser or NetSupport School?
IDs are:
Lockdown Browser: fogjeanjfbiombghnmkmmophfeccjdki
NetSupport School Student: gcjpefhffmcgplgklffgbebganmhffje

**Comment by SlitheringLizard1** [comment]
if you have inspect, then possibly
